content/applications/hr/referrals.rst

```diff
@@ -730,7 +746,7 @@ Click :guilabel:`Create` and a blank alert form loads. Enter the following infor
   the alert should be displayed for, select the company from the drop-down menu. If this field
   remains blank, the alert is visible to everyone with access to the Referrals application. If a
   company is specified, only user's within that company (who also have access to the Referrals
-  application) will see the alert.
+  application) will see the alert. This field only appears when in a multi-company environment.
```

Comment:
```suggestion
  application) see the alert. This field **only** appears when in a multi-company database.
```
Changed environment to database to keep consistent with how it was written earlier - but choose which one you like better :-)